src/main/java/io/pinecone/model/IndexStatus.java

```diff
@@ -0,0 +1,63 @@
+package io.pinecone.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
**question:** Is this needed on `IndexMetaDatabase` and `IndexMeta` as well?

Reply:
Discussed internally on why it's needed.